src/view/drag-handle/drag-handle-types.js

```diff
@@ -1,3 +1,5 @@
+// @flow
```

Comment:
`// @flow` was missing from the top of the file. Adding it made things actually start validating correctly around the place